server.js

```diff
@@ -32,18 +32,19 @@ module.exports = class Server {
   }
 
   initMiddlewares() {
-    this.server.use(morgan('tiny'));
+    this.server.use(morgan("tiny"));
     this.server.use(express.json());
     this.server.use(cors({ origin: configEnv.allowedOrigin }));
   }
 
   initRoutes() {
-    this.server.use('/', () => {
-      throw new ApiError(404, 'Not found', {
-        message: 'Not authorized',
-      });
-    }); //express.static(path.join(__dirname, 'public')));
-    // this.server.use('/api/users', usersRouter);
+    // this.server.use('/', () => {
```

Comment:
This middleware should be placed at the bottom if you actually want to do something like that